### DISCUSSION TITLE
How to make it work with .liquid file? [Prettier Plugin]

### DISCUSSION THREAD

**Discussion by jcsuzanne** [tailwindlabs/tailwindcss#12390] [Help]
Hello,

I want to use Tailwind inside a Shopify project with Prettier formatting.
I'm using the official Shopify prettier plugin https://shopify.dev/docs/themes/tools/liquid-prettier-plugin and it seems to be compatible with the Tailwind Prettier library https://github.com/tailwindlabs/prettier-plugin-tailwindcss

When i save my file, the HTML/Liquid tags are formatted but the class attributes with tailwind classes are never re-ordered.

Can you help?

Using (in package.json)
```
"@shopify/prettier-plugin-liquid": "^1.3.4",
"prettier": "^3.0.3",
"prettier-plugin-tailwindcss": "^0.5.7"
```

(In .prettierrc)
```
{
  "overrides": [
    {
      "files": "*.js",
      "options": {
        "parser": "flow"
      }
    }
  ],
  "printWidth": 80,
  "tabWidth": 2,
  "singleQuote": true,
  "pluginSearchDirs": false,
  "twigAlwaysBreakObjects": false,
  "bracketSameLine": true,
  "twigFollowOfficialCodingStandards": false,
  "plugins": ["@shopify/prettier-plugin-liquid", "prettier-plugin-tailwindcss"]
}
```

**Comment by Velkamhell97** [comment]
Did you find any solution?

**Comment by jS5t3r** [comment]
Maybe this: https://github.com/Shopify/prettier-plugin-liquid
